lib/elixir/test/elixir/process_test.exs

```diff
@@ -9,7 +9,8 @@ defmodule ProcessTest do
     assert Process.put(:foo, :bar) == nil
     assert Process.put(:foo, :baz) == :bar
 
-    assert Process.get_keys() == [:foo]
+    # `:rand_seed` is injected by ExUnit during the execution
+    assert Process.get_keys() == [:rand_seed, :foo]
```

Comment:
I had to add this to the tests, do you think that we should add an option like  `ExUnit.configure seed_type: :global`, with default value for `:individual`?